src/tests/unit/tests/injected/element-based-view-model-creator.test.ts

```diff
@@ -180,11 +200,10 @@ describe('ElementBasedViewModelCreator', () => {
             id: 'some other id',
         } as DecoratedAxeNodeResult;
 
-        const scanResultStoreData = {
-            results: [unifiedResultOne, unifiedResultTwo],
-            rules: unifiedRules,
-            platformInfo: null,
-        } as UnifiedScanResultStoreData;
+        const ScanNodeResults = [
```

Comment:
Same comment as previous two regarding casing: 
```suggestion
        const scanNodeResults = [
```